packs/elastic/actions/search.py

```diff
@@ -0,0 +1,87 @@
+from easydict import EasyDict
+from lib.items_selector import ItemsSelector
+from lib.elastic_action import ElasticAction
+from collections import defaultdict
+import logging
+import sys
+import elasticsearch
+import json
+
+logger = logging.getLogger(__name__)
+
+
+class SearchRunner(ElasticAction):
+
+    def __init__(self, config=None):
+        super(SearchRunner, self).__init__(config=config)
+        self._iselector = None
+
+
+    @property
+    def iselector(self):
+        """
+        Used to fetch indices/snapshots and apply filter to them.
+        """
+        if not self._iselector:
+            self._iselector = ItemsSelector(self.client, **self.config)
+        return self._iselector
+
+
+    def run(self, **kwargs):
```

Comment:
Can you please explicitly declare method signature here? It looks like this class is only used by one action so it should be possible to do that without any other code changes.